app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <application
         android:name=".MifosSelfServiceApp"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_icon"
```

Comment:
@Pritom14 I can not merge this PR right know. I have to talk to the community regarding the App Logo. Maybe someone designed the logo. 

Put this PR on hold for now.